docs/security-reviews/README.md

```diff
@@ -28,5 +28,6 @@ Please see the report for the specific details.
 | 2024-08 | Cantina              | Fault proof MIPS: `MIPS.sol`                                                                                                                                                                                              | [./2024_08_Fault-Proofs-MIPS_Cantina.pdf](./2024_08_Fault-Proofs-MIPS_Cantina.pdf)                                                                         | 71b93116738ee98c9f8713b1a5dfe626ce06c1b2     | op-contracts/v1.4.0 |
 | 2024-08 | Spearbit             | Fault proof no-MIPS: All contracts in the `packages/contracts-bedrock/src/dispute` directory                                                                                                                              | [./2024_08_Fault-Proofs-No-MIPS_Spearbit.pdf](./2024_08_Fault-Proofs-No-MIPS_Spearbit.pdf)                                                                 | 1f7081798ce2d49b8643514663d10681cb853a3d     | op-contracts/v1.6.0 |
 | 2024-10 | 3Doc Security        | Fault proof MIPS: `MIPS.sol`                                                                                                                                                                                              | [./2024_10-Cannon-FGETFD-3DocSecurity.md](./2024_10-Cannon-FGETFD-3DocSecurity.md)                                                                         | 52d0e60c16498ad4efec8798e3fc1b36b13f46a2     | op-contracts/v1.8.0 |
+| 2025-01 | Spearbit             | Multi-thread & 64-bit Cannon                                                                                                                                                                                              | [2025_01-MT-Cannon-Spearbit.pdf](./2025_01-MT-Cannon-Spearbit.pdf)                                                                                         | cc2715c3d6ebef374451b598f48980ad817e0a0e     | op-contracts/v1.6.0 |
```

Comment:
The last column is titled "Subsequent Release" - which I think implies this should be the release that includes the audited changes.  Since we haven't released these changes yet, I think it should be blank for now: 
```suggestion
| 2025-01 | Spearbit             | Multi-thread & 64-bit Cannon                                                                                                                                                                                              | [2025_01-MT-Cannon-Spearbit.pdf](./2025_01-MT-Cannon-Spearbit.pdf)                                                                                         | cc2715c3d6ebef374451b598f48980ad817e0a0e     |  |
```